src/docfx/build/page/BuildPage.cs

```diff
@@ -266,14 +262,14 @@ private static JObject LoadSchemaDocument(ErrorBuilder errors, Context context,
             var schema = context.TemplateEngine.GetSchema(mime);
             var pageModel = (JObject)context.JsonSchemaTransformer.TransformContent(errors, schema, file, validatedObj);
 
-            if (context.Config.OutputType == OutputType.PageJson && TemplateEngine.IsLandingData(mime))
+            if (TemplateEngine.IsLandingData(mime))
             {
                 var landingData = JsonUtility.ToObject<LandingData>(errors, pageModel);
                 var razorHtml = RazorTemplate.Render(mime, landingData).GetAwaiter().GetResult();
 
                 pageModel = JsonUtility.ToJObject(new ConceptualModel
                 {
-                    Conceptual = ProcessHtml(context, file, razorHtml),
+                    Conceptual = ProcessHtml(context, file, razorHtml, skipValidation: true),
```

Comment:
The behavior is different from the original, we can just always apply bookmark validation for `LandingData`